pkg/virt-handler/vm.go

```diff
@@ -226,6 +189,73 @@ func MapPersistentVolumes(vm *v1.VM, restClient cache.Getter, namespace string)
 	return vmCopy, nil
 }
 
+func (d *VMHandlerDispatch) processVmUpdate(vm *v1.VM, exists bool) error {
```

Comment:
How would you feel about reversing the tense of the "exists" boolean passed into this function? I think it might convey more meaning as a function parameter if it were named something along the lines of "delete" or "shouldDelete" etc...